Gemfile

```
@@ -10,13 +10,16 @@ gem 'rails-controller-testing'
 spree_opts = if ENV['SPREE_PATH']
                 { 'path': ENV['SPREE_PATH'] }
              else
-                { 'github': 'spree/spree', 'branch': 'main', 'glob': 'backend/engines/**/*.gemspec' }
+                { 'github': 'spree/spree', 'branch': 'main', 'glob': 'spree/**/*.gemspec' }
```

RuboCop (1.84.2)

[convention] Use 2 (not 3) spaces for indentation.

(Layout/IndentationWidth)